be/src/exec/workgroup/work_group.h

```diff
@@ -210,10 +252,17 @@ class WorkGroupManager {
     double get_cpu_unadjusted_actual_use_ratio() const;
 
     void log_cpu();
+    void apply(const std::vector<TWorkGroupOp>& ops);
+    std::vector<TWorkGroup> list_workgroups();
 
 private:
-    std::mutex _mutex;
-    std::unordered_map<int, WorkGroupPtr> _workgroups;
+    void create_workgroup_unlocked(const WorkGroupPtr& wg);
+    void alter_workgroup_unlocked(const WorkGroupPtr& wg);
+    void delete_workgroup_unlocked(const WorkGroupPtr& wg);
```

Comment:
Are these three methods guarded by write lock of `_mutex`? Maybe we could add some comments.

Reply:
ok